cylc/rose/__init__.py

```diff
@@ -14,9 +14,48 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 """
+
+Cylc Rose
+=========
+
 Cylc Rose is the bridge between Rose suite configurations and Cylc workflows.
 
 Cylc Rose is a replacement for the ``rose suite-run`` command (present in Rose
 versions 2019.01 and earlier). It reads the ``rose-suite.conf`` file and
 performs the required actions.
+
+Pre install
+===========
+
+Before running ``cylc install`` or ``cylc reinstall``  Cylc Rose will:
+
+- Load any rose configuration defined in the installation source directory.
+- Export any environment variables set in the ``[env]`` section
+  of the configuration.
+- Return the configuration so that Cylc can access template variables.
+
+Post install
+============
+
+After running ``cylc install`` or ``cylc reinstall`` Cylc Rose will:
+
+- Copy a ``rose-suite.conf`` file if it is in the source directory.
```

Comment:
This is me nitpicking (feel free to ignore)...
I think this could be a tad ambiguous, I can see users thinking copy it to...?

Reply:
NNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNNo ot at all I agree

Reply:
VVVVVVVVVDI much?